### DISCUSSION TITLE
Django timeout, will FASTAPI help in this context?

### DISCUSSION THREAD

**Discussion by scheung38** [fastapi/fastapi#6639] [Questions] [answered]
Could someone comment if timeout in Django due to large data and slow bandwidth context be alleviated by async await in FASTAPI?

**Comment by raphaelauv** [comment, marked as answer]
this place is for fastAPI issues , go to https://github.com/tiangolo/fastapi/discussions or https://gitter.im/tiangolo/fastapi

could you close your issue , thanks

**Comment by tiangolo** [comment]
Thanks for the help here @raphaelauv ! 👏  🙇 

@scheung38  it would probably be better to ask in a new GitHub issue, following the template that will guide you to structure the question in a way that makes it easier to understand your intention and need, to be able to help you better. And also providing a self-contained, [minimal, reproducible, example](https://stackoverflow.com/help/minimal-reproducible-example) that I can copy-paste to replicate it.

Given that I'll close this one.

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
